### DISCUSSION TITLE
Preparing datasets with CP2K for virial data

### DISCUSSION THREAD

**Discussion by shuaijiang-ustc** [deepmodeling/deepmd-kit#1524] [Q&A]
Hi, I am preparing datasets for deepmd with CP2K. The systems are gas phase molecules, so the static DFT calculations are made with PBC. The calculations of energy and forces are pretty straghtforward, however, I am not sure how to calculate virial. There is STRESS_TENSOR keyword in CP2K, is this the right way to prepare virial data? Many thanks.

**Comment by wanghan-iapcm** [comment]
You may use https://github.com/deepmodeling/dpdata to convert cp2k output to deepmd-kit training data.

**Reply by shuaijiang-ustc** [reply]
Many thanks!
